src/test/java/Homework17.java

```diff
@@ -0,0 +1,36 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.junit.Assert;
+
+// ...
+
+@Test
+public void addSongToPlaylist() {
+
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--remote-allow-origins=*");
+
+        WebDriver driver = new ChromeDriver(options);
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+        String expectedUrl = "https://qa.koel.app/";
+        driver.get(expectedUrl);
+
+        //i am skipping the login part because its redundant
+        //starting from searching
+        driver.findElement(By.xpath("//*[@id="searchForm"]/input")).sendKeys("Episode 2");
+        driver.findElement(By.xpath("//*[@id="searchExcerptsWrapper"]/div/div/section[1]/h1/button")).click();
+        driver.findElement(By.xpath("//*[@id="songResultsWrapper"]/div/div/div[1]/table/tr/td[1]")).click();
```

Comment:
Declare WebElement variables for the elements like search input, search button, song link, addToButton, favoritesOption, and the notification element.

For example : 
```
WebElement searchInput = driver.findElement(By.xpath("//*[@id='searchForm']/input"));
        searchInput.sendKeys("Episode 2");
```

In your code, you had issues with using double quotes within a string that is enclosed by double quotes. This caused syntax errors
"//*[@id="searchForm"]/input"))
Replace the inner double quotes with single quote to fix the problem.